sql/synth_recipes.sql

```diff
@@ -3597,25 +3597,25 @@ INSERT INTO `synth_recipes` VALUES (59928,1,0,0,0,0,0,0,7,0,0,4100,4242,13442,0,
+INSERT INTO `synth_recipes` VALUES (59940,1,0,0,0,0,0,0,85,0,0,4100,4242,17849,0,0,0,0,0,0,0,2157,2157,2171,2171,1,1,1,1,'Hellish Bugle (desynth)');  -- 2021.11.01
 INSERT INTO `synth_recipes` VALUES (59941,1,0,0,0,0,0,0,80,0,0,4100,4242,15211,0,0,0,0,0,0,0,887,887,792,793,1,1,1,1,'Reraise Hairpin (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59942,1,0,0,0,0,0,0,255,0,0,4100,4242,13108,0,0,0,0,0,0,0,850,887,887,887,1,1,1,1,'Coral Gorget (desynth)');
+INSERT INTO `synth_recipes` VALUES (59942,1,0,0,0,0,0,0,79,0,0,4100,4242,13108,0,0,0,0,0,0,0,850,850,887,887,1,1,1,1,'Coral Gorget (desynth)');  -- 2021.11.01
 INSERT INTO `synth_recipes` VALUES (59943,1,0,0,0,0,0,0,70,0,0,4100,4242,13464,0,0,0,0,0,0,0,864,864,902,902,1,1,1,1,'Demon\'s Ring (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59944,1,0,0,0,0,0,0,255,0,0,4100,4242,12436,0,0,0,0,0,0,0,850,866,866,866,1,2,2,2,'Dragon Mask (desynth)');
+INSERT INTO `synth_recipes` VALUES (59944,1,0,0,0,0,0,0,87,0,0,4100,4242,12436,0,0,0,0,0,0,0,850,850,866,866,1,1,1,2,'Dragon Mask (desynth)');  -- 2021.11.01 XI and allakhazam
 INSERT INTO `synth_recipes` VALUES (59945,1,0,0,0,0,0,0,60,0,0,4100,4242,13458,0,0,0,0,0,0,0,896,896,896,896,1,1,1,1,'Scorpion Ring (desynth)');
 INSERT INTO `synth_recipes` VALUES (59946,1,0,0,0,0,0,0,52,0,0,4100,4242,17257,0,0,0,0,0,0,0,893,893,893,893,1,1,1,1,'Bandit\'s Gun (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59947,1,0,0,0,0,0,0,255,0,0,4100,4242,12437,0,0,0,0,0,0,0,850,1587,1586,1586,1,1,1,1,'Gavial Mask (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59948,1,0,0,0,0,0,0,255,0,0,4100,4242,14426,0,0,0,0,0,0,0,816,816,816,816,1,1,1,1,'Shade Harness (desynth)');
+INSERT INTO `synth_recipes` VALUES (59947,1,0,0,0,0,0,0,100,0,0,4100,4242,12437,0,0,0,0,0,0,0,850,1587,1586,1586,1,1,1,1,'Gavial Mask (desynth)');  -- 2021.11.01
+INSERT INTO `synth_recipes` VALUES (59948,1,0,0,0,0,0,0,255,0,0,4100,4242,14426,0,0,0,0,0,0,0,816,816,816,816,1,1,1,1,'Shade Harness (desynth)');  -- 2021.11.01 https://ffxi.allakhazam.com/dyn/guilds/dBonecraft.html
 INSERT INTO `synth_recipes` VALUES (59949,1,0,0,0,0,0,0,57,0,0,4100,4242,16794,0,0,0,0,0,0,0,817,882,893,893,3,1,1,1,'Bone Scythe (desynth)');
-INSERT INTO `synth_recipes` VALUES (59950,1,0,0,0,0,0,0,37,0,0,4100,4242,13459,0,0,0,0,0,0,0,864,895,895,895,1,1,1,1,'Horn Ring (desynth)'); -- based on synth lvl 10/31/21
--- INSERT INTO `synth_recipes` VALUES (59951,1,0,0,255,0,0,0,255,0,0,4100,4242,16409,0,0,0,0,0,0,0,649,880,880,855,1,2,3,1,'Lynx Baghnakhs (desynth)'); -- no level data
+INSERT INTO `synth_recipes` VALUES (59950,1,0,0,0,0,0,0,37,0,0,4100,4242,13459,0,0,0,0,0,0,0,864,895,895,895,1,1,1,1,'Horn Ring (desynth)');  -- 2021.11.01
```

Comment:
Looks like only change for Horn Ring is the comment, I am guessing this s a result of you and I fixing things last night so it got overwritten. The previous comment is preferred.

Reply:
I will update as requested.

Reply:
Looks like we finally wound up with some weird duplication.  The previous Horn Ring entry was from one of my PR's the other day (around 10/31/21). Noticed a few other recent ones got disabled again.  I am guessing it is because the branch wasn't resynched before submitting?  I've got them marked up in my xls sheet and can get them set back after this goes through and I resynch branches again.

Reply:
Sounds good.